include/crpropa/massDistribution/Ferriere.h

```diff
@@ -26,12 +26,12 @@ class Ferriere: public Density {
 
 public:
 	/** Coordinate transformation for the CentralMolecularZone region. Rotation arround z-axis such that X is the major axis and Y is the minor axis
-	@param postion position in galactic coordinates with Earth at (-8.5kpc, 0, 0)
+	@param position position in galactic coordinates with Earth at (-8.5kpc, 0, 0)
 	@return position in local coordinates for the CMZ region
 	*/
 	Vector3d CMZTrafo(const Vector3d &position) const;
```

Comment:
This is the first time I'm looking at this code.
Maybe `Transformation` instead of `Trafo` would have been a better name, especially considering that "trafo" is not an internationally used term.

Reply:
Yes, I will change the name.